cishouseholds/impute.py

```diff
@@ -166,12 +166,12 @@ def fill_forward_only_to_nulls_in_dataset_based_on_column(
     df
     id
     date
-    visit_type
```

Comment:
This docstring doesn't line up with the actual parameters above. I'd delete the "Parameters" section if it doesn't add any documentation for the inputs